conbench/templates/index.html

```diff
@@ -25,16 +25,16 @@ <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
 <div class="row">
     <div class="col-md-12">
         <table id="runs" class="table table-striped table-bordered table-hover">
-          <caption>Runs</div>
+          <caption>Runs
           </caption>
             <thead>
                 <tr>
                     <th scope="col">Run</th>
                     <th scope="col">Author</th>
                     <th scope="col">Repository</th>
-                    <th scope="col">Commit</th>
-                    <th scope="col">Hash</th>
                     <th scope="col">Hardware</th>
+                    <th scope="col">Commit hash</th>
```

Comment:
I'd say let's just call this `Commit`. Then the header is as wide as the column content, and it's still precise.